principles/index.html

```diff
@@ -15,6 +15,13 @@
           companyURL: 'https://nytimes.com/',
           url: 'https://berjon.com/',
           w3cid: 34327,
+        },
+                {
+          name: 'Lukasz Olejnik',
+          company: 'Independent researcher (Invited Expert)',
+          companyURL: 'https://lukaszolejnik.com.com/',
```

Comment:
I think there's an extra ".com"?
```suggestion
          companyURL: 'https://lukaszolejnik.com/',
```